optimum/habana/accelerate/accelerator.py

```diff
@@ -366,8 +368,10 @@ def prepare_model(self, model: torch.nn.Module, device_placement: bool = None, e
         if self.state.dynamo_plugin.backend != DynamoBackend.NO and not is_compiled_module(model):
             compile_kwargs = self.state.dynamo_plugin.to_kwargs()
```

Comment:
Ideally would be to put our setting directly into state's kwarg. Not sure if it is possible though